dependencyManagement/build.gradle.kts

```diff
@@ -65,6 +65,9 @@ val CORE_DEPENDENCIES = listOf(
   "io.opentelemetry.javaagent:opentelemetry-javaagent-tooling:${otelInstrumentationAlphaVersion}",
   // temporarily overriding transitive dependency from azure-core until next azure-bom release
   // which targets at least reactor-netty-http:1.1.1
+  // fix CVE-2023-52428 https://github.com/advisories/GHSA-gvpg-vgmx-xg6w
+  // TODO (heya) remove this once azure-identity is updated in April
+  "com.microsoft.azure:msal4j:1.14.3",
   "io.projectreactor.netty:reactor-netty-http:1.1.17",
```

Comment:
looks like the new lines split the above comment from it's target

Reply:
it was from a merge conflict.

Reply:
it's currently using io.projectreactor.netty:reactor-netty-http:**1.1.15**, should we remove the comment above? 

```
// temporarily overriding transitive dependency from azure-core until next azure-bom release
  // which targets at least reactor-netty-http:1.1.1
```

Reply:
I think it would be ok to remove both the comment and the pinned dependency version